server/src/main/java/org/candlepin/dto/api/v1/OwnerInfoDTOTranslator.java

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright (c) 2009 - 2020 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.dto.api.v1;
+
+import org.candlepin.dto.ModelTranslator;
+import org.candlepin.dto.ObjectTranslator;
+import org.candlepin.model.OwnerInfo;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * The OwnerInfoDTOTranslator provides translation from
+ * OwnerInfo model objects to OwnerInfoDTO
+ */
+public class OwnerInfoDTOTranslator implements ObjectTranslator<OwnerInfo, OwnerInfoDTO> {
```

Comment:
Let's use this as an opportunity to rename OwnerInfo to *not* that. OwnerStatistics would be a better fit here

Reply:
It looks like there's only 1 endpoint that uses OwnerInfo as output, and ideally we wouldn't need a translator (we'd just change `OwnerInfoCurator.getByOwner` to populate the OwnerInfoDTO that is defined in yaml), but the problem is that OwnerInfo is not _just_ a DTO, and has a lot of `add*()` methods that contain logic (and yaml-defined DTOs can't be easily altered to contain additional methods with extra logic).

One solution about eliminating the translator entirely would be to use the builder pattern here, but creating a `OwnerInfoBuilder` that will own all these `add*()` methods and accumulate the data, and when we're done we'd call `.build()` which would return an OwnerInfoDTO.

Reply:
Updated the code to build OwnerInfo from OwnerInfoBuilder.

Reply:
ack from me